src/js/utils/parser.ts

```diff
@@ -1,14 +1,16 @@
-import { exists } from 'utils/validator';
-import { isNaN } from 'utils/underscore';
+import { exists } from './validator';
+import { isNaN } from './underscore';
 
 // Returns the absolute file path based on a relative filepath, and optional base path
-export function getAbsolutePath(path, base) {
-    if (!exists(base)) {
+export function getAbsolutePath(path: string): string;
+export function getAbsolutePath(path: string, base: string): string;
+export function getAbsolutePath(path: string, base?: string): string {
+    if (!base || !exists(base)) {
```

Comment:
Overloading methods.
I know this is small, but idk if we want to space them out. Maybe it's nicer on the eyes.

Reply:
Why?

Reply:
Makes sense. I'm still just stuck in that mindset because I like overloading. Lol